### PR TITLE
DO NOT MERGE Build each commit test

### PR DESCRIPTION
**Pull request by mheon**
Just testing some things

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -341,7 +341,7 @@ validate: gofmt .gitvalidation validate.completions
 
 build-all-new-commits:
 	# Validate that all the commits build on top of $(GIT_BASE_BRANCH)
-	git rebase $(GIT_BASE_BRANCH) -x make
+	git rebase $(GIT_BASE_BRANCH) -x "make | awk '{ print strftime(\"%H:%M:%S\"), $0 }'"
```
Comment (edsantiago) [inline Makefile]: You can safely drop the double quotes

Comment (edsantiago) [reply Makefile]: As in: the outside ones; then the inside ones (%H:etc) don't need to be backslashed

Comment (edsantiago) [reply Makefile]: Grumble. You have to double-dollar it: `$$0`. (I thought `make` would respect the single-quotes; it doesn't)

Comment (mheon) [reply Makefile]: Without outside quotes, we get one time print for the entire git rebase command - is that what you intended?

Comment (edsantiago) [reply Makefile]: `awk` filters line by line, so each line gets its own timestamp (assuming I/O gets flushed properly on this system)

**Comment by cevich** [comment]
From IRC:

[17:50] <podbot_52358441> Cirrus-CI testing successful for PR #2848: https://cirrus-ci.com/build/5070118894698496
[17:53] <cevich> mheon: ^^^ that's you.   Let's do these disk/cpu/mem changes (minus the extra timig info - I'm working on that for all tasks in #2849)

**Comment by mheon** [comment]
Closing in favor of https://github.com/containers/libpod/pull/2853

**Pull request closed without merging**